compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/parser/BallerinaParser.java

```diff
@@ -18179,6 +18180,21 @@ private STNode getExpression(STNode ambiguousNode) {
                         errorCons.typeReference, errorCons.openParenToken, errorArgs, errorCons.closeParenToken);
             case IDENTIFIER_TOKEN:
                 return STNodeFactory.createSimpleNameReferenceNode(ambiguousNode);
+            case INDEXED_EXPRESSION:
+                STNodeList keys = (STNodeList) ((STIndexedExpressionNode) ambiguousNode).keyExpression;
```

Comment:
Since `(STIndexedExpressionNode) ambiguousNode` is repeated, shall we extract it to a separate variable?

Reply:
I'll change it.